onewallet/src/main/java/org/kinecosystem/onewallet/IOneWalletClient.kt

```diff
@@ -0,0 +1,15 @@
+package org.kinecosystem.onewallet
+
+import android.app.Activity
+import android.content.Intent
+import kin.sdk.KinAccount
+
+interface IOneWalletClient {
+
+    fun onActivityCreated(activity: Activity, kinAccount: KinAccount, requestCode: Int,
+                          actionButtonResId: Int, progressBarResId: Int)
+
+    fun onActivityResult(activity: Activity, resultCode: Int, intent: Intent)
+
+    fun onActivityDestroyed()
```

Comment:
should consider additional interface that works with fragments

Reply:
consider adding a different method for binding the views, you can have it as your View classes instead of resId, and give the dev the freedom to get their view whenever they need (on fragment creation/ activity creation).
in any case, there's should be a method for `onActivityCreated` to instantiate things on restarts

Reply:
Thanks I did that